tests/storage/exception_test.py

```diff
@@ -77,6 +84,28 @@ def test_info():
         }
 
 
+def test_VolumeGroupDoesNotExist():
+    # Require a VG name or UUID at initialization.
+    # Empty constructor shall raise.
+    with pytest.raises(ValueError):
+        e = storage_exception.VolumeGroupDoesNotExist()
+
+    # Expected error type is LVMCommandError.
+    with pytest.raises(TypeError):
+        e = storage_exception.VolumeGroupDoesNotExist("vg-name", error="error")
+
+    # Correct initialization.
+    fake_error = storage_exception.LVMCommandError(
+        rc=5, cmd=["fake"], out=["fake output"], err=["fake error"])
+    e = storage_exception.VolumeGroupDoesNotExist("vg-name", error=fake_error)
+    assert e.error == fake_error
+    # Check error format
+    formatted = str(e)
+    assert "vg_name=vg-name" in formatted
+    assert f'error={fake_error}'.replace('\'', r'\'') in formatted
```

Comment:
This is a little bit ugly and fragile in case python changes the way it format quoted string, but I don't have a better way.

Reply:
A little, yes. If that gives us issues in the midterm, we can change the check to something else.